framework/test/unit/modules/dpos_v2/module.spec.ts

```diff
@@ -1192,6 +1181,8 @@ describe('DPoS module', () => {
 						missedMoreThan1Block.some(missedForger => missedForger.equals(delegateAddress))
 					) {
 						expect(currentDelegate.consecutiveMissedBlocks).toBe(2);
+					} else if (delegateAddress.equals(lastForgerAddress)) {
```

Comment:
same question as above

Reply:
Logic of this test was broken since `lastForgerAddress` also misses 1 block, I omitted start and end slots of the range in my new commit([d8c1a8d](https://github.com/LiskHQ/lisk-sdk/pull/7082/commits/d8c1a8db69694350d1425353e60bc15d7c158a6d)), and fixed the logic, please see the commit.